jekyll_prototype/_config.yml

```diff
@@ -14,3 +14,5 @@ exclude:
   - CNAME
   - README.md
   - LICENSE.md
+host: 0.0.0.0
```

Comment:
Please remove this. We don't want to serve it globally by default and I'm already used to the default 4000 port.